### ISSUE TITLE
[High] Chrome+JAWS: Keyboard: 'Change default skin tone' control is not accessible using the keyboard.

### ISSUE THREAD

**Issue by SumitDiyora** [Expensify/App#9669]
## Action Performed:
1. Using JAWS+ Chrome, open URL staging.new.expensify.com
2. Navigate to the already created room from the Chat list using the Tab key and press Enter on it.
3. Navigate to the Emoji button located at the bottom of the page, inside the textfield.
4. Navigate to Emoji control using Tab key and press Enter to activate it.
5. A pop-up will appear.
6. Navigate to the Change default skin tone control. 
7. Observe 'Change default skin tone' control are accessible using the keyboard or screen reader users..

## Expected Result:
Keyboard should be accessible for 'Change default skin tone' control.

## Actual Result:
'Change default skin tone' control can be activated using a mouse but it's not activated using the keyboard.

## Workaround:
Yes

## Area issue was found in:
Emoji pop-up

## Failed WCAG checkpoints
2.1.1

## User impact:
Keyboard only users will not be able to navigate to the controls via TAB or SHIFT+TAB keys

## Suggested resolution:
Ensure that all controls provide keyboard focus and are operable without the use of a mouse. It is recommended to use native HTML controls such as instead of custom controls as they provide keyboard focus and visible outline focus by default. If custom controls must be used, tabindex="0" can be provided for focus.

## Platform:
- Web
- Desktop App

**Version Number:*v1.1.79-9*
**Reproducible in staging?:*Yes*
**Reproducible in production?:*Yes*
**Issue reported by:*Sumit*

![change default](https://user-images.githubusercontent.com/98817464/176920899-64ca5ca9-70f4-4fa4-91a3-507463e1eaa5.jpg)
